q0_linac.py

```diff
@@ -854,6 +854,6 @@ def waitForLL(self, desiredLiquidLevel=q0_utils.MAX_DS_LL):
         print("downstream liquid level at required value.")
 
 
-Q0_CRYOMODULES: Dict[str, Q0Cryomodule] = CryoDict(
-    cryomoduleClass=Q0Cryomodule, cavityClass=Q0Cavity
+Q0_CRYOMODULES = Machine(
+    cryomodule_class=Q0Cryomodule, cavity_class=Q0Cavity
```

Comment:
I think if we wanted to keep this as a dictionary of cryomodules, we can do `Q0_CRYOMODULES = Machine(cryomodule_class=Q0Cryomodule, cavity_class=Q0Cavity).cryomodules`